libminifi/include/core/repository/VolatileContentRepository.h

```diff
@@ -17,113 +17,35 @@
 
 #pragma once
 
-#include <map>
-#include <memory>
-#include <string>
-#include <string_view>
-
-#include "AtomicRepoEntries.h"
-#include "io/AtomicEntryStream.h"
 #include "core/ContentRepository.h"
-#include "properties/Configure.h"
-#include "core/Connectable.h"
-#include "core/logging/LoggerFactory.h"
-#include "utils/GeneralUtils.h"
-#include "VolatileRepositoryData.h"
-#include "utils/Literals.h"
+#include <map>
```

Comment:
This caused the linter error
```suggestion
#include <unordered_map>
#include <string>
#include <memory>
#include <mutex>

#include "core/ContentRepository.h"
```